.github/workflows/build_ubuntu.yml

```diff
@@ -0,0 +1,92 @@
+name: Build Ubuntu
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Info
+      run: echo GITHUB_WORKSPACE=$GITHUB_WORKSPACE
+      
+    - name: Apt update
+      run: sudo apt-get update
+
+    - name: Install tools
+      run: sudo apt-get -y install git cmake build-essential gettext help2man openssl wget #clang-3.8
+ 
+    - name: Prepare sources for act (run actions local)
+      run: echo #rm -r 3rdparty/aubio 3rdparty/ced
```

Comment:
If i'm correct this doesn't need to be done since we build within a container. However if something gets cached it's better to run `git clean -ffdx` to clear out all non-repo files.